tools/BatchModelGenerator/BatchModelGenerator.csproj

```diff
@@ -4,7 +4,7 @@
   <Import Project="$(RepoTools)Common.Netcore.Dependencies.targets" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
No sure whether will cause compatibility issue, @isra-fel , could you help to check it?

Reply:
This is a tool used during development. I don't see any problem here.